cmd/juju/caas/gke.go

```diff
@@ -184,9 +185,6 @@ func (g *gke) listClusters(account, project, region string) (map[string]cluster,
 	cmd := []string{
 		"gcloud", "container", "clusters", "list", "--filter", "status:RUNNING", "--account", account, "--project", project, "--format", "value\\(name,zone\\)",
 	}
-	if region != "" {
```

Comment:
removing this means that we are no longer filtering only those clusters in the specified region; we will get all clusters in any region